tests/bouncer/util_test.py

```diff
@@ -16,184 +16,96 @@ def test_parse_document_raises_if_annotated_deleted():
+    quote = util.parse_document(es_annotation_doc)["quote"]
     assert quote == "Hypothesis annotation for example.com"
 
 
-def test_parse_document_returns_text():
-    text = util.parse_document({
-        "_id": "annotation_id",
-        "_source": {
-            "target": [{
-                "source": "http://example.com/example.html",
-                "selector": [{}]
-            }],
-            "group": "__world__",
-            "shared": True,
-            "text": "test_text"
-        }
-    })["text"]
-
+def test_parse_document_returns_text(es_annotation_doc):
+    es_annotation_doc["_source"]["text"] = "test_text"
+    text = util.parse_document(es_annotation_doc)["text"]
     assert text == "test_text"
 
 
-def test_parse_document_returns_boilerplate_when_no_text():
-    text = util.parse_document({
-        "_id": "annotation_id",
-        "_source": {
-            "target": [{
-                "source": "http://example.com/example.html",
-                "selector": [{}]
-            }],
-            "group": "__world__",
-            "shared": True,
-            "text": ""
-        }
-    })["text"]
-
+def test_parse_document_returns_boilerplate_when_no_text(es_annotation_doc):
+    text = util.parse_document(es_annotation_doc)["text"]
     assert text == util.ANNOTATION_BOILERPLATE_TEXT
 
 
-def test_parse_document_returns_show_metadata_true_when_shared_and_world():
-    show_metadata = util.parse_document({
-        "_id": "annotation_id",
-        "_source": {
-            "target": [{
-                "source": "http://example.com/example.html",
-                "selector": [{}]
-            }],
-            "group": "__world__",
-            "shared": True
-        }
-    })["show_metadata"]
-
+def test_parse_document_returns_show_metadata_true_when_shared_and_world(es_annotation_doc):
+    show_metadata = util.parse_document(es_annotation_doc)["show_metadata"]
     assert show_metadata is True
 
 
-def test_parse_document_returns_document_uri_from_web_uri_when_pdf():
-    document_uri = util.parse_document({
-        "_id": "annotation_id",
-        "_source": {
-            "target": [{
-                "source": "urn:x-pdf:the-fingerprint",
-                "selector": []
-            }],
-            "group": "__world__",
-            "shared": True,
-            "document": {"web_uri": "http://example.com/foo.pdf"}
-        }
-    })["document_uri"]
+def test_parse_document_returns_document_uri_from_web_uri_when_pdf(es_annotation_doc):
+    es_annotation_doc["_source"]["target"][0]["source"] = "urn:x-pdf:the-fingerprint"
+    es_annotation_doc["_source"]["document"] = {"web_uri": "http://example.com/foo.pdf"}
+
+    document_uri = util.parse_document(es_annotation_doc)["document_uri"]
 
     assert document_uri == "http://example.com/foo.pdf"
 
 
-def test_parse_document_raises_when_uri_from_web_uri_not_string_for_pdfs():
+def test_parse_document_raises_when_uri_from_web_uri_not_string_for_pdfs(es_annotation_doc):
```

Comment:
This test case seems to be testing with significantly different input than it did before; is that intentional? If so, why?

Before:

```python
{
    "_id": "annotation_id",
    "_source": {
        "target": [{
            "source": "urn:x-pdf:the-fingerprint",
            "selector": []
        }],
        "group": "__world__",
        "shared": True,
        "document": {"web_uri": 52}
    }
}
```

After:

```python
{
    "_id": "annotation_id",
    "_source": {
        "target": [{
            "source": 52,
            "selector": [{}]
        }],
        "group": "__world__",
        "shared": True,
    },
    "document": {"web_uri": "http://example.com/foo.pdf"}
}
```

Reply:
Gah! This was a mistake, albeit with the same outcome. The `52` should have been assigned to the "web_uri" field.